src/redux/actions/cats.js

```diff
@@ -1 +1,2 @@
 export const onCat = () => ({ type: 'cats/RANDOM_CAT' })
+export const adoptCat = () => ({ type: 'cats/ADOPT_CAT' })
```

Comment:
This function needs to take the cat you are adopting, and send it as the payload of the action